Source/Core/Core/IOS/IPC.cpp

```diff
@@ -109,6 +406,48 @@ static void SDIO_EventNotify_CPUThread(u64 userdata, s64 cycles_late)
     device->EventNotify();
 }
 
+bool SetupMemory(u64 ios_version)
+{
+  auto target_imv = std::find_if(
+      ios_memory_values.begin(), ios_memory_values.end(),
+      [&](const IosMemoryValues& imv) { return imv.ios_number == (ios_version & 0xffff); });
+
+  if (target_imv == ios_memory_values.end())
+  {
+    ERROR_LOG(IOS, "Unknown IOS version: %x", ios_version & 0xffffffff);
```

Comment:
I changed it to just print the entire title_id, and I think it's more natural to show this in hex.